Fix #24161 Again

Aurum22 opened the pull request:
Much like OUTDATED, this will fix monkeys throwing augments.
It also adds the ability to scan for augments with the body scanner.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

🆑 Cronac
bugfix: Monkeys are no longer ripping out their augments and throwing them across the room.
tweak: Augments now show up on medical scanners and related admin scans.
/🆑

Datraen committed:
Xilvuxix Tweaks Round 3

Adds energy machetes, ion thrusters, and liquidfood for the culinary challenged skrell; clarifies names on doors on the vessel; fixes jumpsuit icon.

CakeQ committed:
Increases explorer count to 5

Lowers the prospector count to 2

Updates explo closet counts

comma committed:
Changes how area test exception lists are formed

Objective is threefold:
-Adding exception to each map's unit test file is fucking pain
-Maintaining that stuff is fucking pain
-Lists are getting out of control, in awe of their size

Adds another list of exceptions, area_coherency_test_exempted_root_areas, which works as it's named. Mostly added it for exoplanets, so don't need to bother adding every single new exoplanet type.

Main change is making map templates take care of their own exceptions.
Now they have lists set up similar to main map datums, and update map datum's list on their New

Higgin committed:
Leaves machete damage alone.

Fixes busted proc.

Prevents material weapons from taking damage on parry except against other material weapons.
Increases material weapon health.
Reduces crowbar and toolbox accuracy penalties.
Increases machete damage.
Adds an inferior, printable, steel machete to the autolathe.

Aurum22 committed:
Fixes Morgue Tray Occlusion

HeyBanditoz committed:
Flying mobs are now immune to lava.

Aurum22 committed:
Cleans up Morgue Fixes

So fresh so clean...

halworsen committed:
Push data after switching firing mechanism firemode

halworsen committed:
Cap pulling claw weight class by assembly weight class

halworsen committed:
Fix #24063

halworsen committed:
Fix #23764

halworsen committed:
Fix #23043

halworsen committed:
Weight class restriction changes to throwers

ghostsheet committed:
Adding Ore Box to Supply crate

Aurum22 committed:
Merge pull request #1 from Baystation12/dev

Baymerge 1

comma committed:
Charon and Guppy now only need Basic skill for overmap flight

Makes piloting difficulty skill set per ship, sets it to Basic for Charon and Guppy.

afterthought2 committed:
May fix #11517.

WatermelonsEverywhere committed:
Adds a spare explorer equipment crate to supply

afterthought2 committed:
Fixes #22470.

afterthought2 committed:
Fixes #24893.

afterthought2 committed:
Tweaks to acids; closes #19568.

Aurum22 committed:
Augments and Apes

Fixes monkeys throwing augments around, makes augments scannable.

Spookerton committed:
extinguisher refilling clarifications

:cl:
bugfix: Refilling a fire extinguisher from a tank no longer tells you the extinguisher is full when the tank is empty.
/:cl:

Also macrofies nearby messages.

comma commented on code/modules/organs/external/_external.dm:
```diff
@@ -1430,6 +1430,11 @@ obj/item/organ/external/proc/remove_clamps()
 		if(unknown_body)
 			. += "Unknown body present"
+	if(internal_organs.len)
+		for(var/A in internal_organs)
```
can just do 
```
for(var/obj/item/organ/internal/augment/aug in internal_organs)
```
don't need to check internal_organs.len then too, loop wills safely handle empty stuff

comma commented:
Travis fail is legit
```
[07:00:00] Runtime in monkey.dm,84: pick() from empty list
  proc name: handle npc (/datum/species/monkey/handle_npc)
  src: Stok (/datum/species/monkey/unathi)
  call stack:
  Stok (/datum/species/monkey/unathi): handle npc(the unknown (/mob/living/carbon/human))
  the unknown (/mob/living/carbon/human): Life(20, 0, Mobs (/datum/controller/subsystem/processing/mobs))
  Mobs (/datum/controller/subsystem/processing/mobs): fire(1)
  Mobs (/datum/controller/subsystem/processing/mobs): ignite(1)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing(0)
```

afterthought2 committed:
Merge pull request #25139 from Spookerton/fix-extinguishers

extinguisher refilling clarifications

PsiOmegaDelta committed:
Automatic changelog generation for PR #25139 [ci skip]

afterthought2 committed:
Merge pull request #25133 from WatermelonsEverywhere/explorerscelebrate

Adds a spare explorer equipment crate to supply - Take Two

afterthought2 committed:
Merge pull request #25122 from HeyBanditoz/we_didnt_start_the_fire

Flying mobs are now immune to lava.

afterthought2 committed:
Merge pull request #25115 from comma/unithackery

Changes how area test exception lists are formed

afterthought2 committed:
Merge pull request #25120 from Aurum22/MorgueFix

Fix #25110

PsiOmegaDelta committed:
Automatic changelog generation for PR #25120 [ci skip]

Aurum22 committed:
Code Cleaning

Aurum22 committed:
Travis Change

Going on a gut feeling here.

Aurum22 committed:
Appeasing Travis

PsiOmegaDelta committed:
Automatic changelog update [ci skip]

comma committed:
Merge pull request #25137 from afterthought2/bughunt

Couple of fixes; nerfs acids.

PsiOmegaDelta committed:
Automatic changelog generation for PR #25137 [ci skip]

comma committed:
Merge pull request #25125 from ghostsheet/Ore-Boxing

Adding Ore box to Supply crate

PsiOmegaDelta committed:
Automatic changelog generation for PR #25125 [ci skip]

comma committed:
Merge pull request #25124 from halworsen/icmisc

Tweaks and runtime fixes for ICs

comma committed:
Merge pull request #25039 from Datraen/XilvuxixTweaksNumeroTres

Xilvuxix Tweaks Round 3

comma committed:
Merge pull request #25111 from Higgin/ghetto-warfare

Various melee weapon tweaks.

PsiOmegaDelta committed:
Automatic changelog generation for PR #25111 [ci skip]

comma committed:
Merge pull request #25074 from CakeQ/moreSplorers

Increases explorer slot count, reduces prospector slot count.

PsiOmegaDelta committed:
Automatic changelog generation for PR #25074 [ci skip]

PsiOmegaDelta committed:
Automatic changelog update [ci skip]

comma committed:
Adds couple dept goals for exoploration

Plant scanning and flag planting

comma committed:
Splits scanners.dm

No changes, just moved all scanners into different files.
Preparation for changing scanners a bit, that file was pain to navigate (looking at you health scanner)

comma committed:
Changed way goal departments are instancized

Now if map doesn't have a list set, it'll instancize all possible depts. Cleared default list so 'all depratments' is default behaviour.
Less lists to keep track of when you're adding stuff, and it mirrors current behaviour anyway since all existing depts were in the list.

xales committed:
Merge pull request #25159 from comma/scanners

Splits scanners.dm

afterthought2 committed:
Merge pull request #25131 from comma/ezpz

Charon and Guppy now only need Basic skill for overmap flight

PurpleMartinJCK committed:
Merge pull request #25158 from comma/whatdo

Adds couple dept goals for exoploration

Aurum22 committed:
Monkey and Augment Changes

Aurum22 committed:
Merge branch 'MonkeyIsland' of https://github.com/Aurum22/AurumBayFixes into MonkeyIsland

Aurum22 committed:
Devmerge

Aurum22 committed:
Merge branch 'MonkeyIsland' of https://github.com/Aurum22/AurumBayFixes into MonkeyIsland

Aurum22 committed:
Devmerge 2

Aurum22 committed:
Merge branch 'MonkeyIsland' of https://github.com/Aurum22/AurumBayFixes into MonkeyIsland

Aurum22 committed:
devmerge 3

Aurum22 committed:
Merge branch 'MonkeyIsland' of https://github.com/Aurum22/AurumBayFixes into MonkeyIsland

Aurum22 committed:
Merge branch 'MonkeyIsland' of https://github.com/Aurum22/AurumBayFixes into MonkeyIsland

Aurum22 committed:
Revert "Merge branch 'MonkeyIsland' of https://github.com/Aurum22/AurumBayFixes into MonkeyIsland"

This reverts commit 5208199d9b9d675fec8d0aec7a37f0416a0488e8, reversing
changes made to e7dff9df7c6c78921abdc4a9b718edb2abe17f6c.

Aurum22 commented:
Went off of the rails with trying to squash. Going to fix now.
